README.md

```diff
@@ -42,6 +42,9 @@ npm run build
 ```
 
 ### Test
+
+Install docker-compose
+
```

Comment:
I think we need to tell people to actually run the docker-compose, right?
```suggestion
Install docker-compose

```bash
docker compose -f tests/setUpIpfs/docker-compose.yml
```

Reply:
I am not sure. docker-compose should be available right after installation. Or do you mean to start ipfs-cluster container? It is started in script https://github.com/energywebfoundation/ew-did-registry/blob/c81b0398da329cd463d007544c209d62321a2411/tests/setUpIpfs/index.ts#L15